### PR TITLE
[Merged by Bors] - Add bors configuration

### PR DESCRIPTION
**Pull request by nicholastmosher**
This PR is intended to be merged after https://github.com/infinyon/fluvio/pull/937

@sehz if we want to use this, you'll need to add the Bors bot to this repository. You can do that here: https://bors.tech/ at the top you click "Log into dashboard". It will ask to add Bors to your Github account, but you can selectively choose which repositories it has access to, so you can choose `infinyon/fluvio`. Once that's done, I believe we should be able to use the bors commands in this PR.

Here's the command documentation for how to use Bors: https://bors.tech/documentation/. The general gist though is just that if you are reviewing a PR, instead of clicking the "Merge/Rebase/Squash" button, you just comment with `bors r+` and it will take care of adding it to a queue to be merged, then it will merge it once all CI passes.

### CODE REVIEW

**Comment by sehz** [comment]
Nice

File: bors.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+status = [
+    "Rustfmt (ubuntu-latest, stable)",
+    "Clippy (ubuntu-latest, stable)",
+    "Unit tests (ubuntu-latest, stable)",
```
Comment (sehz) [inline bors.toml:4]: Is these correct names?

Comment (nicholastmosher) [reply bors.toml:4]: I believe yes. I tested out naming conventions on my own fork. If you use bors + it will not touch master unless they are corret

**Comment by sehz** [comment]
bors +

**Comment by sehz** [comment]
r+

**Comment by nicholastmosher** [comment]
bors try

**Comment by nicholastmosher** [comment]
bors try-

**Review by sehz — APPROVED** [review]

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**